components/browser/engine-gecko-nightly/src/test/java/mozilla/components/browser/engine/gecko/webextension/GeckoWebExtensionTest.kt

```diff
@@ -40,6 +40,7 @@ import org.mozilla.geckoview.WebExtension
 import org.mozilla.geckoview.WebExtensionController
 
 @RunWith(AndroidJUnit4::class)
+@Suppress("Deprecation")
```

Comment:
I would prefer if we only put that on the methods that use `registerWebExtension`. It's easier to see what's deprecated and we can address it soon.